htdocs/product/reassort.php

```diff
@@ -323,32 +343,6 @@
 		print '</div>';
 	}
 
-	$param = '';
```

Comment:
Are you sure we can remove this ? Don't we need it for the 
print_liste_field_titre("Ref", $_SERVER["PHP_SELF"], "p.ref", '', $param, "", $sortfield, $sortorder);
later ?